### PR TITLE
Purge windows cache

### PR DESCRIPTION
**Pull request by cocreature**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -46,6 +46,8 @@ function bazel() {
 # which is a workaround for this problem.
 bazel shutdown
 
+bazel clean --expunge
```
Comment (garyverhaegen-da) [inline build.ps1]: should that say
```
`-`-expunge
```
?

Comment (cocreature) [reply build.ps1]: pretty sure this worked the last couple of times, don’t ask me why

**Comment by cocreature** [comment]
> Nope.

Rude @stefanobaghino-da

File: build.ps1 (hunk 1 above)
Comment (garyverhaegen-da) [reply build.ps1]: So... Not asking why it works without the backticks, as requested, but do you have any idea why the `bazel build` commands on lines 53 & 57 need them, then?

Comment (cocreature) [reply build.ps1]: tbh, no. I don’t remember the error we ran into without those escapes but at least for expunge they don’t seem to be necessary.

**Comment by cocreature** [comment]
looks like another PR being merged has forced a cache refresh so this should no longer be necessary.

**Pull request closed without merging**